### PR TITLE
Add pyPEG2 recipe

### PR DESCRIPTION
**Pull request by v0lat1le**

### CODE REVIEW

**Comment by v0lat1le** [comment]
@conda-forge/core anything else I need to do for this to go through?

File: recipes/pyPEG2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "pyPEG2" %}
+{% set version = "2.15.2" %}
+{% set md5 = "84057d292808553290f0b78f42c64bbe" %}
```
Comment (jakirkham) [inline recipes/pyPEG2/meta.yaml]: Please use `sha256` from [pypi.org]( https://pypi.org/ ).

File: recipes/pyPEG2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - lxml
```
Comment (jakirkham) [inline recipes/pyPEG2/meta.yaml]: Is this needed at build time or only at run time?

**Comment by v0lat1le** [comment]
@jakirkham good points, ty! All fixed up now.

**Comment by jakirkham** [comment]
Thanks @v0lat1le.

**Pull request merged**